aws/s3.tf

```diff
@@ -5,11 +5,20 @@ resource "aws_s3_bucket" "tiered_storage" {
   force_destroy = var.allow_force_destroy
 }
 
-resource "aws_s3_bucket_acl" "tiered_storage" {
-  count  = var.tiered_storage_enabled ? 1 : 0
-  bucket = aws_s3_bucket.tiered_storage[count.index].id
-  acl    = "private"
-}
+## https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html
+#resource "aws_s3_bucket_acl" "tiered_storage" {
+#  count  = var.tiered_storage_enabled ? 1 : 0
+#  bucket = aws_s3_bucket.tiered_storage[count.index].id
+#  acl    = "private"
+#}
+#
+#resource "aws_s3_bucket_ownership_controls" "s3_bucket_acl_ownership" {
+#  for_each = toset(aws_s3_bucket.tiered_storage[*].id)
+#  bucket   = each.key
+#  rule {
+#    object_ownership = "ObjectWriter"
+#  }
+#}
```

Comment:
[question] Is this intended?

Reply:
Yes, the commented docs explain why in the warning doc. Basically ACL + IAM == error and not recommended.